test/sol5/kyberNetwork.js

```diff
@@ -184,16 +182,78 @@ contract('KyberNetwork', function(accounts) {
                 add: true
             });
         });
-          
-            //TODO: RemoveReserveFromNetwork
-            //TODO: ListReservePairs
-            //TODO: DAOContractSet 
-            //TODO: KyberNetworkParamsSet
-            //TODO: KyberNetworkSetEnable
-            //TODO: KyberProxyAdded
-            //TODO: KyberProxyRemoved
-            //TODO: HandlePlatformFee
-            //TODO: KyberTrade
+        
```

Comment:
can we add describe for this
like reserve listing / removing
So its easier to test only this. or remove only this part.

Reply:
Added it after testing contract events